test/unit/org/apache/cassandra/locator/ReplicationStrategyEndpointCacheTest.java

```diff
@@ -57,7 +54,7 @@ public void setup(Class stratClass, Map<String, String> strategyOptions) throws
         tmd.updateNormalToken(new BigIntegerToken(String.valueOf(20)), InetAddressAndPort.getByName("127.0.0.2"));
         tmd.updateNormalToken(new BigIntegerToken(String.valueOf(30)), InetAddressAndPort.getByName("127.0.0.3"));
         tmd.updateNormalToken(new BigIntegerToken(String.valueOf(40)), InetAddressAndPort.getByName("127.0.0.4"));
-        //tmd.updateNormalToken(new BigIntegerToken(String.valueOf(50)), InetAddressAndPort.getByName("127.0.0.5", null, null));
+        tmd.updateNormalToken(new BigIntegerToken(String.valueOf(50)), InetAddressAndPort.getByName("127.0.0.5"));
```

Comment:
Do you know why was this commented? By the way, this block could be (perhaps) simplified to:
```java
for (int i = 1; i <= 8; i++)
    tmd.updateNormalToken(new BigIntegerToken(String.valueOf(i * 10)), 
                                              InetAddressAndPort.getByName("127.0.0." + i));
```